Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs

```diff
@@ -214,6 +254,83 @@ public async Task GetById_RepositoryThrows_Status500InternalServerErrorReturned(
     }
     #endregion
 
+    #region Create tests
+    [Test]
+    public async Task Create_RepositoryInsertsOnce_CreatedAtActionResultReturned()
+    {
+        var cleaningEvent = CleaningEvents.First();
+        var createRequest = new CleaningEventCreateRequest
+        {
+            PollutedLocationId = PollutedLocations.First().Id,
+            StartTime = cleaningEvent.StartTime,
+            Notes = cleaningEvent.Notes
+        };
+
+        var actionResult = await _controller.Create(createRequest);
+
+        _repository.Verify(r => r.InsertAsync(It.IsAny<CleaningEvent>()), Times.Once);
+
+        Assert.That(actionResult.Result, Is.TypeOf<CreatedAtActionResult>());
+        var result = actionResult.Result as CreatedAtActionResult;
+
+        Assert.That(result, Is.Not.Null);
+        Assert.That(result.StatusCode, Is.EqualTo(StatusCodes.Status201Created));
+
+        Assert.That(result.Value, Is.Not.Null.And.TypeOf<CleaningEventResponse>());
+        var resultLocation = result.Value as CleaningEventResponse;
+        Assert.That(resultLocation, Is.Not.Null);
+        Assert.Multiple(() =>
+        {
+            Assert.That(resultLocation.PollutedLocationId, expression: Is.EqualTo(createRequest.PollutedLocationId));
+            Assert.That(resultLocation.StartTime, Is.EqualTo(createRequest.StartTime));
+            Assert.That(resultLocation.Notes, Is.EqualTo(createRequest.Notes));
+        });
+    }
+
+    [Test]
+    public async Task Create_RepositoryThrows_NotExistingPollutedLocationIdProvided_Status500InternalServerErrorReturned()
```

Comment:
```suggestion
    public async Task Create_RepositoryThrows_Status500InternalServerErrorReturned()
```
the error you're trying to replicate doesn't necessarily originate from pollutedLocationId not existing in the db. Suggesting this change not only because of that, but as was described in the previous review, where I touched on the implementation and not tests, I've specified, that it is not proper behavior that we have there currently.